gammapy/spectrum/models.py

```diff
@@ -1040,6 +1041,75 @@ def evaluate(energy, amplitude, reference, ecut, index_1, index_2):
         return pwl * cutoff
 
 
+class PLSuperExpCutoff4FGL(SpectralModel):
+    r"""Spectral super exponential cutoff power-law model used for 4FGL.
+
+    This model parametrisation is very similar, but slightly different from
+    `PLSuperExpCutoff3FGL` or `ExponentialCutoffPowerLaw3FGL`
+
+    .. math::
+        \phi(E) = \phi_0 \cdot \left(\frac{E}{E_0}\right)^{-\Gamma_1}
+                  \exp \left( a \left(E_0 ^{\Gamma_2} - \E^{\Gamma_2} \right)
+                              \right)
+
+    Parameters
+    ----------
+    index_1 : `~astropy.units.Quantity`
+        :math:`\Gamma_1`
+    index_2 : `~astropy.units.Quantity`
+        :math:`\Gamma_2`
+    amplitude : `~astropy.units.Quantity`
+        :math:`\phi_0`
+    reference : `~astropy.units.Quantity`
+        :math:`E_0`
+    expfactor : `~astropy.units.Quantity`
+        :math: 'a'
```

Comment:
Formatting error, put  backticks like this:
```
:math:`a`
```

Reply:
Done!